wiki/Ranking_Criteria/en.md

```diff
@@ -195,10 +195,11 @@ This category contains explicit allowance statements of concepts and rules that
 
 - **The audio file of a beatmap must...**
   - **...use the `.mp3` or `.ogg` file format.**
-  - **...have an average bit rate no greater than 192 kbps.**
+  - **...have an average bit rate no greater than 192 kbps for `.mp3` files, or 208 kbps for `.ogg` files.**
   - **...have an average bit rate no lower than 128 kbps**, if such a source exists. Otherwise, use the highest quality available.
   - **...not be encoded upwards from a lower bitrate.**
 - **A beatmap may only contain one song file used by all difficulties.** Multiple song files within a single beatmap are unsupported and result in unexpected behaviour with preview times, metadata, etc.
+- **A beatmap of 2 songs combined together must follow the minimum spread requirements corresponding to the length of the longer song.**[^songs-comp-note] This is to avoid artificial extension of songs as an evasion of time limitations in the beatmap section of these criteria. Exceptions can be made for songs that were exclusively released together.
```

Comment:
shouldn't this just be in the beatmap section?

Reply:
hm, initially went with audio because of the songs comp thing but beatmap might make more sense considering the current wording and how it's more oriented about "spread". feel free to adjust at your own discretion